client/src/common/utils/localStorage.js

```diff
@@ -0,0 +1,22 @@
+export const loadSettings = () => {
+  try {
+    const serializedState = localStorage.getItem('settings');
+
+    if (serializedState === null) {
+      return undefined;
+    }
+
+    return JSON.parse(serializedState);
```

Comment:
change to single line:
`return serializedState === null ? undefined : JSON.parse(serializedState)`
or just one if:
```
if (serializedState !== null) {
    return JSON.parse(serializedState);
}
```